crates/aptos-telemetry-service/src/custom_event.rs

```diff
@@ -59,18 +68,20 @@ pub async fn handle_custom_event(
         body.timestamp_micros
             .as_str()
             .parse::<u64>()
-            .map_err(|_| reject::reject())?,
+            .map_err(|_| ServiceError::bad_request("unable to parse timestamp"))?,
     );
 
-    let row = json!({
-        "event_name": telemetry_event.name,
-        "event_timestamp": duration.as_secs(),
-        "event_params": event_params,
-    });
+    let row = BigQueryRow {
+        event_identity: EventIdentity::from(claims),
+        event_name: telemetry_event.name.clone(),
+        event_timestamp: duration.as_secs(),
+        event_params,
+    };
 
-    insert_request
-        .add_row(None, &row)
-        .map_err(|_| reject::reject())?;
+    insert_request.add_row(None, &row).map_err(|e| {
+        error!("unable to create row: {}", e);
+        ServiceError::from(anyhow!("unable to insert row into bigquery"))
```

Comment:
Let's append `e` to the message.

Reply:
Hmm,  I am worried if by including the error, we may end up returning any sensitive information regarding our BQ instance. That's why I logged the error before returning a generic error. This may not be helpful to tie those two events together, so I have a [backlog item](https://linear.app/aptoslabs/issue/ADP-90/log-grouping-with-trace-id-for-cloud-run) to address this via tracing support. wdyt?

Reply:
SGTM.